examples/eks/cluster.yaml

```diff
@@ -16,88 +14,88 @@ spec:
   agentVersion: 3.3.0
   version: 6.1.0
   sysctls:
-  - "fs.aio-max-nr=2097152"
+    - fs.aio-max-nr=2097152
```

Comment:
Iirc yq doesn't support configuring this but could we maybe use a workaround to avoid all these indentations changes in arrays? Especially that it goes against our supposed formatting rule

Reply:
I am not aware of a way to avoid it. It sucks but all the manifests edited with yq have it already.